recipes/nfir/all/patches/0001-find-OpenCV-provided-by-conan_toolchain.cmake.patch

```diff
@@ -0,0 +1,65 @@
+From 6a33009da09e903eb55a22b96de169d43c39fe38 Mon Sep 17 00:00:00 2001
+From: Tomas Krupa <tomas.krupa@innovatrics.com>
+Date: Mon, 24 Feb 2025 11:22:41 +0100
+Subject: [PATCH] find OpenCV provided by conan_toolchain.cmake
+
+---
+ src/lib/CMakeLists.txt | 22 ++++++++++++----------
+ 1 file changed, 12 insertions(+), 10 deletions(-)
+
+diff --git a/src/lib/CMakeLists.txt b/src/lib/CMakeLists.txt
+index 12b3047..e8f87c6 100644
+--- a/src/lib/CMakeLists.txt
++++ b/src/lib/CMakeLists.txt
+@@ -14,24 +14,26 @@ add_library( ${PROJECT_NAME} ${sources} )
+ #   resample_up.cpp
+ # )
+ 
++find_package(OpenCV REQUIRED CONFIG)
++target_link_libraries(${PROJECT_NAME} PRIVATE opencv::opencv)
+ 
+ if(_WIN32_64)
```

Comment:
Simply replace `_WIN32_64` with `FALSE` here. The non-Windows logic is valid for other platforms as well and `/WX` is generally unwanted in public builds anyway.